.github/workflows/CI.yml

```
@@ -1068,4 +1114,4 @@ jobs:
           name: ${{ needs.setup_release.outputs.release_tag }}
           prerelease: true
           tag: ${{ needs.setup_release.outputs.release_tag }}
-          token: ${{ secrets.GH_BOT_TOKEN }}
+          token: ${{ secrets.GH_BOT_TOKEN }}
```

```suggestion
          token: ${{ secrets.GH_BOT_TOKEN }}

```

All files should end with a blank empty line.